make/Main.gmk

```diff
@@ -417,12 +417,22 @@ $(eval $(call SetupTarget, create-source-revision-tracker, \
 ))
 
 BOOTCYCLE_TARGET := product-images
+BOOTCYCLE_SPEC := $(dir $(SPEC))bootcycle-spec.gmk
+
 bootcycle-images:
         ifneq ($(COMPILE_TYPE), cross)
 	  $(call LogWarn, Boot cycle build step 2: Building a new JDK image using previously built image)
 	  $(call MakeDir, $(OUTPUTDIR)/bootcycle-build)
+          # We need to create essential files for the bootcycle spec dir
+	  ( cd $(TOPDIR) && \
+	        $(MAKE) $(MAKE_ARGS) -f make/GenerateFindTests.gmk \
+	         SPEC=$(BOOTCYCLE_SPEC))
```

Comment:
Indentation looks a bit off here.

Reply:
Yeah, it was completely borked. Not only was there an extra space here, but I did not realize the opening parenthesis should set the base indentation level. Fixed now, thanks for noticing!

Reply:
I don't see this as fixed, have you forgot to push a change?